coordinator/src/routes.rs

```diff
@@ -324,16 +330,37 @@ pub async fn post_register(
             "Invalid registration parameters: {register_params:?}"
         )));
     }
-    tracing::info!(?register_params, "Registered new user");
 
     let mut conn = state
         .pool
         .get()
         .map_err(|e| AppError::InternalServerError(format!("Could not get connection: {e:#}")))?;
 
-    if let Some(email) = register_params.email {
+    if let Some(email) = register_params.email.clone() {
+        let waitlist_entry = if let Some(waitlist_entry) = waitlist::with_email(&mut conn, &email)
+            .map_err(|e| {
+            AppError::InternalServerError(format!("could not retrieve waitlist: {e:#}"))
+        })? {
+            tracing::info!("User is on the waitlist");
```

Comment:
This log is not that useful unless you say which user is in the waitlist. Might be a debugging left over to see if the API works?